temporalcli/commands.operator_cluster.go

```diff
@@ -86,3 +87,102 @@ func (c *TemporalOperatorClusterDescribeCommand) run(cctx *CommandContext, args
 		Table:  &printer.TableOptions{},
 	})
 }
+
+func (c *TemporalOperatorClusterListCommand) run(cctx *CommandContext, args []string) error {
+	cl, err := c.Parent.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	var nextPageToken []byte
+	var execsProcessed int
+	for pageIndex := 0; ; pageIndex++ {
```

Comment:
This is one of those cases I wonder if we never expect a large list and therefore should just collect the list and dump all at once. Regardless, I added to #448 to revisit.